### DISCUSSION TITLE
Model in AR suddenly jump up from base position

### DISCUSSION THREAD

**Discussion by NitinJ88** [google/model-viewer#4808] [Q&A]
### Description
Model in AR suddenly jump up from base position and in some cases it disappear. User need to relaunch the AR again to see the model.

I checked and there was similar issue reported but this one look little different.
https://github.com/google/model-viewer/discussions/4355 

#### Live Demo

https://www.dropbox.com/scl/fo/a24mgjxkodh0wc6exm3jt/AAfbgOWQmvTLovakc31YG1Q?rlkey=q6b7gyax2lfgdosqo4w8f7qkx&st=w3v91xfz&dl=0 

### Version

- model-viewer: v3.2.1

### Browser Affected

- [*] Chrome, version: 125.0.6422.142 (Official Build) (64-bit)
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [*] Windows

### AR

- [*] WebXR
- [*] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
This is expected behavior because AR is hard. The jump is when ARCore finally figure out where your floor is (the model is below the floor to that point). It can't find your floor because you're just pointing your camera around your room, but it can only find the floor from parallax, so you need to actually move your camera from side to side, rather than turning it. I'd recommend a prompt to help the user understand this: https://modelviewer.dev/examples/augmentedreality/index.html#webXR
